.github/workflows/build.yml

```diff
@@ -171,7 +143,7 @@ jobs:
           coverage: none
 
       - name: Install dependencies
-        run: composer install
+        run: composer install --no-dev ${{ matrix.php == '8.4' && '--ignore-platform-req=php+' || '' }}
```

Comment:
When installing the dependencies to test the Phar we don't need the dev dependencies. And even if it they are not installed, composer does check them so we need to tell it to ignore the php version requirement or it will fail due to PHPStan 5

Reply:
I think we don't need to install the dependencies at all. We don't use them

Reply:
Absolutely right, removed